app/models/book.rb

```diff
@@ -72,6 +72,7 @@ def bindings_as_tuples
 
   def self.search(q)
     if q
+      q = q.last.present? ? q : q[0...-1]
       Book.where({ status: ["NP", "IP", "OP"] })
```

Comment:
@cdoyle-temple Although it's not part of the original PR, please indent the continuation of line 75/76

Reply:
Should be in a separate Commit. Closing...